### ISSUE TITLE
Deprecate legacy http endpoint

### ISSUE THREAD

**Issue by MarinPostma** [libsql/sqld#683]
The legacy HTTP endpoint has been superseded by the hrana protocol. I propose we remove it for clarity.

**Comment by psarna** [comment]
All of my demos use http 😅 it has this single advantage of being super easy to hand-code in a webpage

**Comment by MarinPostma** [comment]
I agree, I use is all the time for testing as well, and hrana is not very ergonomic, but at the same time, it's a maintenance burden :( Let's think about it

**Comment by penberg** [comment]
AFAICT the Fermyon Spin runtime uses the HTTP endpoint: https://github.com/fermyon/spin/blob/main/crates/sqlite-libsql/src/lib.rs#L12
